api/src/main/java/com/datastrato/gravitino/rel/expressions/partitions/Partition.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.rel.expressions.partitions;
```

Comment:
Why partition is under package "expression"?

Reply:
fixed